themes/beaver/layouts/partials/assets/careers-css-resources.html

```
@@ -1,4 +1,4 @@
-{{/* Careers page CSS bundle composition.
+{{/* Careers pageCSS bundle composition.
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)